SmartDeviceLink/SDLProxy.m

```diff
@@ -42,9 +42,9 @@
 typedef void (^URLSessionDownloadTaskCompletionHandler)(NSURL *location, NSURLResponse *response, NSError *error);
 
 NSString *const SDLProxyVersion = @"4.6.1";
-const float startSessionTime = 10.0;
-const float notifyProxyClosedDelay = 0.1;
-const int POLICIES_CORRELATION_ID = 65535;
```

Comment:
POLICIES_CORRELATION_ID on line no. 702 has not been updated

Reply:
Fixed